hibernate-core/src/test/java/org/hibernate/test/util/StringHelperTest.java

```diff
@@ -21,6 +30,7 @@
 	private static final String BASE_PACKAGE = "org.hibernate";
 	private static final String STRING_HELPER_FQN = "org.hibernate.internal.util.StringHelper";
 	private static final String STRING_HELPER_NAME = StringHelper.unqualify( STRING_HELPER_FQN );
+	@Rule public final Timeout globalTimeout = new Timeout(10000);
```

Comment:
Doesn't like you're using this, so remove it, along with:
```
import org.junit.Rule;
import org.junit.rules.Timeout;
```
above.

Reply:
done